src/relay/backend/te_compiler.cc

```diff
@@ -134,41 +144,78 @@ class TECompilerImpl : public TECompilerNode {
     return mod;
   }
 
+  void AddExterns(IRModule module) {
```

Comment:
The "goal" here is we could eventually delete this and just leave ExternDefinitions in the module?

Reply:
Forgot to comment this helper, sorry. Yeah, very much a stepping stone pending more official extern binding representation (prob new subclass of BaseFuncNode)